clinica/converters/_utils.py

```diff
@@ -94,7 +100,9 @@ def create_participants_df(
     prev_sheet = 0
     index_to_drop = []
     subjects_to_drop = []
+    subjects_list = []
```

Comment:
Here we handle the `bids_ids` parameter of the function:

Ideally, the parameter `bids_ids` would already be a `list[BIDSSubjectID]` and not a `list[str]`. This is something we could improve later, but we can still convert this list of strings to a list of proper objects:

```python
bids_ids: list[BIDSSubjectID] = [
    bids_id_factory(study_name)(bids_id) for bids_id in bids_ids
]
```